src/test/java/hw14.java

```diff
@@ -0,0 +1,28 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class hw14 {
+
+    @Test
+    public static void RegistrationPage () throws InterruptedException {
+        WebDriver driver = new ChromeDriver();
+        try {
```

Comment:
You can: to maximize browser in Selenium, you need to call the maximize() Selenium command to maximize window interface of the driver class.

`driver.manage().window().maximize();`